target_parquet/__init__.py

```diff
@@ -19,20 +19,23 @@
 from enum import Enum
 from multiprocessing import Process, Queue
 
-from .helpers import flatten, flatten_schema
+from .helpers import flatten, flatten_schema, flatten_schema_to_pyarrow_schema
 
 _all__ = ["main"]
 
 LOGGER = singer.get_logger()
 LOGGER.setLevel(os.getenv("LOGGER_LEVEL", "INFO"))
 
 
-def create_dataframe(list_dict, remove_empty_columns=False):
+def create_dataframe(list_dict, schema, remove_empty_columns=False, force_output_schema_cast=False):

```

Comment:
I think we should remove `remove_empty_columns`.
Also, I don't see the need of `force_output_schema_cast` parameter. It should be always normal behavior.

Reply:
> I think we should remove remove_empty_columns.

Ok!

> Also, I don't see the need of force_output_schema_cast parameter. It should be always normal behavior.

But this could break other's applications that are using the datatype autodetection (when we send the PR to the official repo).